qiskit_superstaq/daily_integration_test.py

```diff
@@ -116,9 +116,8 @@ def test_cq_compile(provider: qss.SuperstaQProvider) -> None:
     circuit = qiskit.QuantumCircuit(1)
     circuit.h(0)
     expected = qiskit.QuantumCircuit(6)
```

Comment:
Something like this maybe?
```suggestion
    assert isinstance(provider.cq_compile(circuit).circuit, cirq.Circuit)
    circuits = provider.cq_compile([circuit]).circuits
    assert len(circuits) == 1 and isinstance(circuits[0], cirq.Circuit)
    circuits = provider.cq_compile([circuit, circuit]).circuits
    assert len(circuits) == 2 and isinstance(circuits[0], cirq.Circuit) and isinstance(circuits[1], cirq.Circuit)
```

Reply:
Yep.